packages/dbus/build.sh

```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://dbus.freedesktop.org
 TERMUX_PKG_DESCRIPTION="Freedesktop.org message bus system"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="Leonid Pliushch <leonid.pliushch@gmail.com>"
-TERMUX_PKG_VERSION=1.12.20
-TERMUX_PKG_REVISION=3
-TERMUX_PKG_SRCURL="https://dbus.freedesktop.org/releases/dbus/dbus-$TERMUX_PKG_VERSION.tar.gz"
-TERMUX_PKG_SHA256=f77620140ecb4cdc67f37fb444f8a6bea70b5b6461f12f1cbe2cec60fa7de5fe
+TERMUX_PKG_VERSION=1.13.18
+TERMUX_PKG_REVISION=4
```

Comment:
No need to bump revision. The line can be removed when increasing TERMUX_PKG_VERSION